union/core/core.py

```diff
@@ -0,0 +1,126 @@
+import inspect
+import io
+import sys
+import traceback
+
+import aiohttp
+import discord
+from discord.ext import commands
+
+
+class Core:
+
+    """
+    Core features of the bot.
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.session = aiohttp.ClientSession(loop=bot.loop)
```

Comment:
This is technically under discussion in #16. Can someone explain the actual benefits/drawbacks of either approach on that issue?